_scripts/instruction-widget/templates/getting-started/sharedhost.html

```diff
@@ -1,13 +1,42 @@
-<p>
-    Do you have the requirements listed above?
-</p>
+<aside class="note">
+  <div class="note-header">
+    <h3>Important Note</h3>
+  </div>
+  <p>These instructions do not apply to people using a <strong>Virtual Private Server (VPS)</strong>; you’ll have to manually manage your Certbot installation, and select your Operating System from the dropdown above.</p>
+</aside>
 
-<h3>Yes, I have these requirements.</h3>
-<p>
-    If you have the requirements, select the operating system that the machine run by the web hosting service is using from the dropdown menu, and follow those instructions.
-</p>
+<h4>Look for your web hosting provider on this list.</h4>
+<p>Are you looking for your web hosting provider, such as Squarespace, Github, or WordPress?</p>
+<p>You might already have HTTPS available and not need Certbot. <a href="hosting_providers/">Find your web hosting provider in our chart of hosting providers.</a></p>
+
+<p class="centered" ><a class="link-button" href="hosting_providers/">find your hosting provider</a><br><br></p>
+
+<p>If your hosting provider has full support for HTTPS: good news! You won’t have to do anything further</p>
+
+<p>If your hosting provider has partial support for HTTPS: good news! You can follow along the linked tutorials for that individual web hosting provider, and get your website on HTTPS without needing Certbot.</p>
+
+<p>If your hosting provider does not support HTTPS: continue to the steps below.</p>
+
+<h4> If your hosting provider does not support HTTPS, the following options are available:</h4>
+
+<p><span class="subhead">You can contact your web hosting provider: tell them you want a free HTTPS certificate through Let’s Encrypt.</span><br>
+You’re probably not the only one using your web hosting provider service who wants HTTPS. You can request that your web hosting provider offer Let’s Encrypt HTTPS certificates as a free part of their hosting package. An effective way to make this ask is through email, their help desk system, or by contacting the web hosting provider through social media.</p>
+
+<p><span class="subhead">You can switch to a different web hosting provider.</span><br>
+Find a web hosting provider who offers full HTTPS support as part of their web hosting package by <a href="/hosting_providers">checking our list.</a></p>
+
+<p><span class="subhead">You might be able to use Certbot.</span><br>
+If you have <a href="/glossary/#ssh">SSH access</a> to the server your website is hosted on, you might be able to use Certbot. You will need to know the <a href="glossary/#software">software</a> and <a href="glossary/#operating-system">system</a> your server is running on. After you confirm the software and system information, you can use the dropdown menus above to generate specific instructions for running Certbot on your server through the command line.</p>
```

Comment:
Seems like the links without at leading "/" are failing the CI checks

Reply:
Thank you @esoterik so much for figuring this out! I managed to fix other broken links after this one, and now they are all fixed!